views/component/providerList/template.njk

```diff
@@ -1,12 +1,11 @@
-<h1 class="govuk-heading-l">Learning providers</h1>
 <ul class="list">
     <li class="provider-headers"><span class="govuk-body first-header">Learning provider name</span><span class="govuk-body">Added on</span></li>
     {% for learningProvider in pageResults.results %}
 
     <li class="list-details plist-details u--border--bgrey">
-        <p class="govuk-body list-item list-name" href="/content-management/course/{{ learningProvider.id }}">{{ learningProvider.name }}</p>
-        <p class="govuk-body list-item list-date">24th Jan 2018</p>
-        <a class="govuk-link list-link " href="#">Edit</a>
+        <p class="govuk-body list-name">{{ learningProvider.name }}</p>
+        <p class="govuk-body list-item list-date">13th August 2018</p>
```

Comment:
As above, should this date be dynamic?

Reply:
Correct, but info wasnt available in Learning catalogue at the time of this branch. Will do later